modules/client-configuration/pages/clients-microos.adoc

```diff
@@ -0,0 +1,126 @@
+[[clients-opensuse-microos]]
+= Registering {opensuse}{nbsp}{microos} Clients
+
+This section contains information about registering clients running these {opensuse}{nbsp}{microos} operating systems:
+
+* {opensuse}{nbsp}{microos}
+
+[WARNING]
+====
+Support for {opensuse}{nbsp}{microos} clients is provided as a technology preview for testing purposes, and not all features are fully functional at this stage.
+This feature is expected to be fully supported in a later version of {productname}.
+Do not use this feature on production systems.
+====
+
+
+
+== Add Software Channels
+
+Before you register {opensuse}{nbsp}{microos} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+The channels you need for this procedure are:
+
+[[opensuse-microos-channels-cli]]
+[cols="1,1,1,1", options="header"]
+.openSUSE MicroOS Channels - CLI
+|===
+
+| OS Version
+| Base Channel
+| Client Channel
+| Updates Channel
+
+| {opensuse}{nbsp}{microos}
+| opensuse_tumbleweed
+| opensuse_tumbleweed-non-oss
+| opensuse_tumbleweed-update
+
+|===
+
+include::snippets/addchannels_novendor_cli.adoc[]
+
+
+
+== Check Synchronization Status
+
+ifeval::[{suma-content} == true]
+
+include::snippets/check_sync_webui_suma.adoc[]
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+endif::[]
+
+include::snippets/check_sync_cli.adoc[]
+
+[IMPORTANT]
+====
+{opensuse}{nbsp}{microos} channels can be very large.
+Synchronization can sometimes take several hours.
+====
+
+
+
+== Trust Certificates Keys on Clients
+
+{opensuse}{nbsp}{microos} is not yet fully enabled, so there are some manual steps to trust the {productname} SSL certificate on {microos} clients.
+
+// Not tested. From https://etherpad.nue.suse.com/p/uyuni-microos. --LKB 2021-04-15
+
+
+.Procedure: Manually Trusting Certificates
```

Comment:
If we are installing and configuring salt... should we use a different name for the procedure?